valhalla/jawn/src/managers/score/ScoreManager.ts

```diff
@@ -0,0 +1,59 @@
+import { dbExecute } from "../../lib/shared/db/dbExecute";
+import { err, Result } from "../../lib/shared/result";
+import { BaseManager } from "../BaseManager";
+
+export class ScoreManager extends BaseManager {
+  public async addScores(
+    requestId: string,
+    scores: Record<string, number>
+  ): Promise<Result<string, string>> {
+    try {
+      const organizationId = this.authParams.organizationId;
+      const scoreKeys = Object.keys(scores);
+      const scoreValues = Object.values(scores);
+
+      const { error: requestError } = await dbExecute(
+        `SELECT id FROM request WHERE id = $1 AND helicone_org_id = $2`,
+        [requestId, organizationId]
+      );
+
+      if (requestError) {
+        return err(`${requestId} not found in organization ${organizationId}`);
+      }
+
+      const organizationIds = Array(scoreKeys.length).fill(organizationId);
+
+      const query = `
```

Comment:
Could we make these 2 calls for brevity? 1 to insert score_attribute, 1 to insert score_values?

Reply:
We can but to inster score_values we need to loop through each score_attribute_id and make db call / or do batch insertion but I like current implementation in 1 db call